docs/VnVPlan/VnVPlan.tex

```diff
@@ -323,64 +323,641 @@ \subsection{Software Validation Plan}
 
-\wss{Subsets of the tests may be in related, so this section is divided into
-  different areas.  If there are no identifiable subsets for the tests, this
-  level of document structure can be removed.}
-
-\wss{Include a blurb here to explain why the subsections below
-  cover the requirements.  References to the SRS would be good here.}
+The subsections below outline tests corresponding to functional 
+requirements in the SRS. Each test is associated with a unique 
+functional area, helping to confirm that the tool meets the 
+specified requirements. Each functional area has its own subsection 
+for clarity.
 
-\subsubsection{Area of Testing1}
+\subsubsection{Code Input Acceptance Tests}
 
-\wss{It would be nice to have a blurb here to explain why the subsections below
-  cover the requirements.  References to the SRS would be good here.  If a section
-  covers tests for input constraints, you should reference the data constraints
-  table in the SRS.}
+This section covers the tests for ensuring the system correctly accepts 
+Python source code files, detects errors in invalid files, and provides 
+suitable feedback (FR 1).
 		
-\paragraph{Title for Test}
+\paragraph{1. Input Acceptance}
+\begin{enumerate}
+\item{Valid Python File Acceptance\\}
+
+Control: Automatic
+					
+Initial State: Tool is idle.
+					
+Input: A valid Python file (filename.py) with valid standard syntax.
+					
+Output: The system accepts the file without errors.
+
+Test Case Derivation: Confirming that the system correctly processes a valid Python file as per FR 1.
+					
+How test will be performed: Feed a syntactically valid .py file to the tool and observe if it’s accepted without issues.
+					
+\item{Feedback for Python File with Bad Syntax\\}
+
+Control: Automatic
+					
+Initial State: Tool is idle.
+					
+Input: A .py file (badSyntax.py) containing deliberate syntax errors that render the file unrunable.
+					
+Output: The system rejects the file and provides an error message detailing the syntax issue.
+
+Test Case Derivation: Verifies the tool’s handling of syntactically invalid Python files to ensure user awareness of the syntax issue, meeting FR 1.
+
+How test will be performed: Feed a .py file with syntax errors to the tool and check that the system identifies it as invalid and produces an appropriate error message.
+
+\item{Feedback for Non-Python File\\}
+
+Control: Automatic
+					
+Initial State: Tool is idle.
+					
+Input: A non-Python file (document.txt) or a file with an incorrect extension (script.js).
+					
+Output: The system rejects the file and provides an error message indicating the invalid file format.
+
+Test Case Derivation: Ensures the tool detects unsupported file types and provides feedback, satisfying FR 1.
+
+How test will be performed: Attempt to load a .txt or other non-Python file, and verify that the system rejects it with a message indicating an invalid file type.
+
+
+\item{Missing Original Test Suite Handling\\}
```

Comment:
Do we also need a test case for when the test suite is correctly provided?
What about when the test suite is for the wrong python file ? -  for this we can have a validation test case to ensure that the test suite passes all tests for the given python code